src/components/input/error.tsx

```diff
@@ -6,7 +6,7 @@ export const Error = (props: Props) => {
   const { className, children, ...restProps } = props;
 
   return (
-    <span {...restProps} className={cn("text-sm text-red-400", className)}>
+    <span {...restProps} className={cn(" text-sm text-red-400", className)}>
```

Comment:
I guess that's unnecessary space here :)
```suggestion
    <span {...restProps} className={cn("text-sm text-red-400", className)}>
```